src/platform/device.gni

```diff
@@ -120,6 +120,11 @@ declare_args() {
   }
 }
 
+declare_args() {
+  # Enable Joint Fabric features
+  chip_device_config_enable_joint_fabric = false
```

Comment:
I had to introduce this in context of https://github.com/project-chip/connectedhomeip/pull/38749 - which has been merged, Need to rebase this PR on latest master.